proposed/clock-meta.md

```diff
@@ -2,21 +2,19 @@
 
 ## 1. Summary
 
-The purpose of using the `ClockInterface` is to provide a standard way to access the system 
-time, that would allow interopability when testing code that relies on the current time 
-rather than relying on installing PHP extensions or use hacks like re-declaring the `time()`
-function in other namespaces. 
+Getting the current time in applications is a rather straight forward thing given the languages native `time()` function (or one of it's similar functionlaities like `utime()` or `new DateTimeImmutable()` etc.)
```

Comment:
s/straight forward/straightforward/ (it's one word)

s/given the languages native/given the language's native/ (make it possessive)

s/one of it's similar/one of its similar/ (make it possessive rather than "it is" contraction)

Reply:
Fixed with f5c2188